# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.h
# Commit: f91e5fbb9520
# Full Hash: f91e5fbb9520301ba4b3da24c22c6f203c1f0af7
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2020-07-11 21:52:53
# Regressor Bug: 1557983
# File Overlap Count: 1
# Description:
#   Bug 1557983. Remove nsPrintJob::HasPrintCallbackCanvas.
#   
#   After refactoring the 'beforeprint'/'afterprint' code, this method is not used
#   any more.
#   
# ==============================================================================

diff -r 965d8227390a -r f91e5fbb9520 layout/printing/nsPrintJob.h
--- a/layout/printing/nsPrintJob.h	Fri Jul 10 18:23:27 2020 +0000
+++ b/layout/printing/nsPrintJob.h	Sat Jul 11 08:47:04 2020 +0000
@@ -131,12 +131,6 @@
 
   void TurnScriptingOn(bool aDoTurnOn);
 
-  /**
-   * Checks to see if the document this print engine is associated with has any
-   * canvases that have a mozPrintCallback.
-   * https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement#Properties
-   */
-  bool HasPrintCallbackCanvas() { return mHasMozPrintCallback; }
   bool PrePrintPage();
   bool PrintPage(nsPrintObject* aPOect, bool& aInRange);
   bool DonePrintingPages(nsPrintObject* aPO, nsresult aResult);
